packages/constants/src/lib/constants/constants.ts

```diff
@@ -738,10 +743,12 @@ export const LIT_NETWORKS: { [key in LitNetwork]: string[] } & {
   localhost: string[];
   internalDev: string[];
 } = {
+
+  // FIXME: Reviewers! These are not used in this branch, scroll down to see remapping logic
   [LitNetwork.Cayenne]: [
-    'https://cayenne.litgateway.com:7370',
-    'https://cayenne.litgateway.com:7371',
-    'https://cayenne.litgateway.com:7372',
+    `${CAYENNE_URL}:7370'`,
+    `${CAYENNE_URL}:7371'`,
+    `${CAYENNE_URL}:7372'`,
```

Comment:
just curious why did we make this change?

Reply:
This array is actually not used anymore, we can remove it. I didn't because there are interfaces/types are relying on it? Need to investigate

Reply:
Ah okay i was thinking it could be removed now. Making a note of this for myself @joshLong145